packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -104,9 +104,9 @@ export default function ShareOptionsMenu({
         squad.active &&
         shareOptions.push({
           icon: squad.image ? (
-            <SquadImage className="mr-2.5 w-5 h-5 text-2xl" {...squad} />
+            <SquadImage className="mr-2.5 w-6 h-6" {...squad} />
           ) : (
-            <MenuIcon Icon={DefaultSquadIcon} />
+            <MenuIcon Icon={DefaultSquadIcon} size="medium" />
```

Comment:
Don't think this adds anything, the `MenuIcon` always renders a icon in medium size :)